universal-application-tool-0.0.1/app/controllers/FileController.java

```diff
@@ -70,7 +71,7 @@ public Result adminShow(Request request, long programId, String fileKey) {
       if (!fileKey.contains(String.format("program-%d", programId))) {
         return notFound();
       }
-      return redirect(amazonS3Client.getPresignedUrl(fileKey).toString());
+      return redirect(storageClient.getPresignedUrl(fileKey, Optional.empty()).toString());
```

Comment:
suggestion (non-blocking): For this line, I'm wondering if it might be cleaner to overload `storageClient.getPresignedUrl` to accept only the fileKey, and then the overload with one argument can call `storageClient.getPresignedUrl` with the empty optional.

Reply:
Done!!